test/indent.txt

```diff
@@ -101,6 +163,13 @@ to foo ; comment
 + ask turtles [ fd 1 ]
 end
 
+Comments3
```

Comment:
Not crazy about this test case. Seems like the kind of thing you would *imagine* people would want rather than something people would *actually* want. If you need to do a multi-line comment, you do it before the line in question.